lib/apr/payments.ex

```diff
@@ -22,6 +22,20 @@ defmodule Apr.Payments do
 
   def payment_info(_, _), do: {:uknown_payment}
 
+  def payment_info_ach(external_id, "payment_intent") do
+    with {:ok, pi} <- PaymentIntent.retrieve(external_id, %{expand: ["payment_method"]}),
```

Comment:
#non-blocking. Looks like payment method is not needed, so we can probably skip expanding it.

Reply:
oh yeah, good point! I can do a followup with that + any other changes we decide to implement based on your review 🙂